incubating/wrappers/s2i/java/Dockerfile.runtime

```diff
@@ -1,4 +1,5 @@
-FROM openjdk:8u131-jre-alpine
+ARG IMAGE_SOURCE=openjdk:8u131-jre-alpine
```

Comment:
Similarly as above, could you also bump the runtime base to `openjdk:13.0.1-jdk-buster` (which is what the latest `core-builder` uses)?